src/cinder/app/cocoa/AppImplMac.mm

```diff
@@ -628,9 +628,21 @@ - (void)resize
 	NSSize nsSize = [mCinderView frame].size;
 	mSize = cinder::ivec2( nsSize.width, nsSize.height );
 
+	NSRect frame = [mWin frame];
+	NSRect content = [mWin contentRectForFrameRect:frame];
+	
+	ivec2 prevPos = mPos;	
+	mPos = ivec2( content.origin.x, [[[NSScreen screens] objectAtIndex:0] frame].size.height - frame.origin.y - content.size.height );
```

Comment:
I don't have a secondary display available at the moment so i'll have to check later. I think you're right, though.